lib/src/Pos/Client/CLI/Params.hs

```diff
@@ -76,6 +77,7 @@ getNodeParams defaultLoggerName cArgs@CommonNodeArgs{..} NodeArgs{..} mGenerated
     npBehaviorConfig <- case behaviorConfigPath of
         Nothing -> pure def
         Just fp -> eitherToThrow =<< liftIO (Yaml.decodeFileEither fp)
+    npFInjects <- mkFInjects cnaFInjects
```

Comment:
I think, this would also deserve a proper and loud warning when not empty. 

```
WARNING: NODE IS RUNNING WITH FAULTY BEHAVIORS ENABLED
```

Reply:
Done!

Reply:
That's how it looks, when `--allow-fault-injection` is passed:
```
Wallet is starting...[cardano-sl.*production*:Info:ThreadId 4] [2018-10-17 21:04:38.84 UTC] using configurations: ConfigurationOptions {cfoFilePath = "/nix/store/sf2gg6k9zcy0066rxd823l1vdw2yd0lv-new-config/configuration.yaml", cfoKey = "default", cfoSystemStart = Just 1539781120000000, cfoSeed = Nothing}
[cardano-sl.*production*:Info:ThreadId 4] [2018-10-17 21:04:39.37 UTC] System start time is 1539781120000000
[cardano-sl.*production*:Info:ThreadId 4] [2018-10-17 21:04:39.37 UTC] Current time is 1539810279372860
[cardano-sl.*production*:Info:ThreadId 4] [2018-10-17 21:04:39.37 UTC] initalizing user secret
[cardano-sl.*production*:Warning:ThreadId 4] [2018-10-17 21:04:39.37 UTC] ***
[cardano-sl.*production*:Warning:ThreadId 4] [2018-10-17 21:04:39.37 UTC] *** FAULT INJECTION MACHINERY ACTIVE, ALL WARRANTIES VOID
[cardano-sl.*production*:Warning:ThreadId 4] [2018-10-17 21:04:39.37 UTC] ***
```